metacat-main/src/main/resources/application.yml

```diff
@@ -17,12 +17,13 @@
 #
 
 management:
-  contextPath: /actuator
   health:
     elasticsearch:
       enabled: false # This is needed for now as we're using a very out of date ES client that throws runtime errors
-  security:
-    enabled: false
+  endpoints:
+    web:
+      exposure:
+        include: "*"
```

Comment:
This might be a little overly aggressive. There are some endpoints like the shutdown and trace ones which are a little dangerous. We turn those off internally.

Reply:
I see. Excluded those (shutdown is disabled by default).